app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,39 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: { class: "form-horizontal"} do |f|
```

Comment:
For `bootstrap_form_for`, try to read documentation on how to set it to form-horizontal. `layout: :horizontal` for instance... 

https://github.com/bootstrap-ruby/rails-bootstrap-forms#horizontal-forms